### PR TITLE
Add schema property to sql database utility class

### PR DESCRIPTION
**Pull request by nfcampos**

### CODE REVIEW

File: langchain/sql_database.py [hunk 1]
```diff
@@ -13,16 +13,18 @@ class SQLDatabase:
     def __init__(
         self,
         engine: Engine,
+        schema: str,
```
Comment (hwchase17) [inline langchain/sql_database.py:16]: can we set default to None?

Comment (nfcampos) [reply langchain/sql_database.py:16]: Good point, done

**Comment by nfcampos** [comment]
Added some new tests too

**Comment by nfcampos** [comment]
Linter should be happy now

**Pull request merged**